### DISCUSSION TITLE
Two aff_logs?

### DISCUSSION THREAD

**Discussion by xlime4** [Monika-After-Story/MonikaModDev#8126] [QA - tech support + general] [answered]
I recently moved my Monika to a new computer. Everything seems to be working well, but I've noticed that there's two aff_log files. Is that normal? Will it affect my game in any way? Thanks for any advice.

**Comment by Booplicate** [comment, marked as answer]
`.txt` is the old format for logs, `.log` is the new one. Keep all logs just in case.

**Reply by xlime4** [reply]
Oh, I see. Thank you very much!
